dashboards/gcp-cloud-spanner/gcp-cloud-spanner.json

```diff
@@ -2096,7 +2069,817 @@
+          "visualization": {
+            "id": "viz.line"
+          },
+          "rawConfiguration": {
+            "facet": {
+              "showOtherSeries": false
+            },
+            "legend": {
+              "enabled": true
+            },
+            "nrqlQueries": [
+              {
+                "accountId": 0,
+                "query": "FROM Metric SELECT sum(database_spanner_read_stats_top_avg_bytes) WHERE instrumentation.source = 'gcpspanner' FACET fingerprint SINCE 60 minutes ago TIMESERIES"
+              }
+            ],
+            "platformOptions": {
+              "ignoreTimeRange": false
+            },
+            "yAxisLeft": {
+              "zero": true
+            }
+          }
+        },
+        {
+          "title": "Average leader refresh delay seconds",
+          "layout": {
+            "column": 1,
+            "row": 7,
+            "width": 4,
+            "height": 3
+          },
+          "linkedEntityGuids": null,
+          "visualization": {
+            "id": "viz.line"
+          },
+          "rawConfiguration": {
+            "facet": {
+              "showOtherSeries": false
+            },
+            "legend": {
+              "enabled": true
+            },
+            "nrqlQueries": [
+              {
+                "accountId": 0,
+                "query": "FROM Metric SELECT latest(database_spanner_read_stats_top_avg_leader_refresh_delay_seconds) WHERE instrumentation.source = 'gcpspanner' FACET fingerprint SINCE 60 minutes ago TIMESERIES"
+              }
+            ],
+            "platformOptions": {
+              "ignoreTimeRange": false
+            },
+            "yAxisLeft": {
+              "zero": true
+            }
+          }
+        },
+        {
+          "title": "Average locking delay seconds",
+          "layout": {
+            "column": 5,
+            "row": 7,
+            "width": 4,
+            "height": 3
+          },
+          "linkedEntityGuids": null,
+          "visualization": {
+            "id": "viz.line"
+          },
+          "rawConfiguration": {
+            "facet": {
+              "showOtherSeries": false
+            },
+            "legend": {
+              "enabled": true
+            },
+            "nrqlQueries": [
+              {
+                "accountId": 0,
+                "query": "FROM Metric SELECT sum(database_spanner_read_stats_top_avg_locking_delay_seconds) WHERE instrumentation.source = 'gcpspanner' FACET fingerprint SINCE 60 minutes ago TIMESERIES"
+              }
+            ],
+            "platformOptions": {
+              "ignoreTimeRange": false
+            },
+            "yAxisLeft": {
+              "zero": true
+            }
+          }
+        }
+      ]
+    }
+  ],
+  "variables": [
+    {
+      "name": "project_id",
+      "items": null,
+      "defaultValue": {
+        "string": ""
+      },
+      "nrqlQuery": {
+        "accountIds": [
+          3462097
```

Comment:
Looks like we have some account Ids here, mind running the [sanitization script](https://github.com/newrelic/newrelic-quickstarts/blob/main/CONTRIBUTING.md#dashboards) / setting these to 0

Reply:
I ran the sanitize script again, so this should be cleaned up.